leader-election-impl/src/test/java/com/palantir/leader/proxy/AwaitingLeadershipProxyTest.java

```diff
@@ -64,6 +65,9 @@
 
     @Rule public final ExpectedException expect = ExpectedException.none();
 
+    @Rule
+    public final RenderTracingRule rule = new RenderTracingRule();
```

Comment:
do you think it's worth doing the golden master thing? my proposal is that it could be, because this codepath is fragile, and we want to make sure that we're not undoing any async work as a result by accident (which is quite hard to test atm). Perhaps we don't need it for this PR, but it's something I've been thinking about with all the partitioning/perf work i.e. how do we avoid regressing things that aren't easily tested?

Reply:
you needn't migrate big bang, that'd be way too much effort, could be a similar policy to assertj i.e. use it moving forward etc. or just for this rule if we think it has value

Reply:
The issue was more bumping the Guava dependency, though we checked internally and determined that this was almost certainly acceptable.

Reply:
Turns out this causes more problems than expected because e.g. of dev breaks within this codebase, so not proceeding with this right now.